liquidator/liquidator.js

```diff
@@ -19,11 +19,13 @@ class Liquidator {
   }
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
-  queryAndLiquidate = async priceFeed => {
+  queryAndLiquidate = async priceFunction => {
+    const contractTime = await this.empContract.methods.getCurrentTime().call();
+    const priceFeed = priceFunction(contractTime);
+
     Logger.debug({
       at: "Liquidator",
-      message: "Checking for under collateralized positions",
-      inputPrice: priceFeed
+      message: "Checking for under collateralized positions"
```

Comment:
Why remove this from the log? You have resolved the price above, would it not be good to still log this in the `debug` message?

Reply:
I removed it to standardize with the Disputer bot, but I think its prob worth adding 👍

Reply:
done

Reply:
Looks good. we should do it in the disputor as well at some point